app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :users
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  has_many :likes
+  has_many :comments
```

Comment:
[OPTIONAL]

- What do you think about adding the `dependent: :destroy` option to the `has_many` associations? This will help to make sure that all associated objects are deleted when a `Post` gets deleted.

You can learn more about this [here](https://guides.rubyonrails.org/association_basics.html#dependent)

Reply:
Thank you! I'll do it!